lib/alterity.rb

```diff
@@ -71,6 +71,11 @@ def execute_alter(table, updates)
       raise("[Alterity] Command failed. Full output: #{result_str}") unless exit_status.success?
     end
 
+    def sanitize(command)
+      return unless config.password.present?
```

Comment:
this will return nil when we still want to have the command printed

Reply:
oops, sorry, fixed